src/components/Inventory/InventoryContainer.scss

```diff
@@ -0,0 +1,98 @@
+@use "../../styles/partials/variables" as *;
+@use "../../styles/partials/mixins" as *;
+
+.card-container {
+  width: 336px;
+  margin: auto;
+  box-shadow: 0px 2px 5px 0px #13182c1a;
+  background: $color-white;
+  position: relative;
+  top: -70px;
+  border-radius: 10px;
+  padding-top: 10px;
+  padding-left: 30px;
+  padding-right: 30px;
+  padding: 10px 30px 0 30px;
+  @include tablet {
+    width: 690px;
+    padding: 10px 0 0 0;
+  }
+
+  &__cta {
+    @include tablet {
+      display: flex;
+      align-items: center;
+      position: absolute;
+      right: 49px;
+      top: 13px;
+    }
+  }
+
+  &__table-heading {
+    display: none;
+    background-color: $color-light-grey;
+
+    @include tablet {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      height: 40px;
+      padding: 0 2rem;
+    }
+  }
+
+  &__heading {
+    @include tablet {
+      padding: 0 2rem;
+    }
+  }
+}
+
+.search {
+  position: relative;
+  width: 328px;
+  height: 40px;
+
+  @include tablet {
+    right: 25px;
+    width: 185px;
+  }
+}
+
+.search__input {
+  width: 100%;
+  padding: 9px;
+  padding-right: 5px; /* Space for the search icon */
```

Comment:
Use shorthands for `padding` so you dont need both these properties. See the docs